doc/python/sankey-diagram.md

```diff
@@ -71,6 +71,13 @@ import urllib, json
 url = 'https://raw.githubusercontent.com/plotly/plotly.js/master/test/image/mocks/sankey_energy.json'
 response = urllib.request.urlopen(url)
 data = json.loads(response.read())
+
+# override gray link colors with 'source' colors
+opacity = 0.4
+data['data'][0]['node']['color'] = ['rgba(255,0,255, 0.8)' if color == "magenta" else color for color in data['data'][0]['node']['color']]
+data['data'][0]['link']['color'] = [data['data'][0]['node']['color'][src] for src in data['data'][0]['link']['source']]
+data['data'][0]['link']['color'] = [color.replace("0.8", str(opacity)) for color in data['data'][0]['link']['color']]
```

Comment:
```suggestion
data['data'][0]['link']['color'] = [data['data'][0]['node']['color'][src].replace("0.8", str(opacity)) 
                                    for src in data['data'][0]['link']['source']]
```

Reply:
setting the link color in just one line could show in a clearer way that you're just taking the source node color and changing its opacity